backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/SqlDatabase.java

```diff
@@ -616,6 +616,24 @@ public DSLContext getJooq() {
     return jooq;
   }
 
+  void getJooqAsAdmin(JooqTransaction transaction) {
+    if (inTx()) {
+      String user = connectionProvider.getActiveUser();
+      try {
+        connectionProvider.setActiveUser(ADMIN_USER);
```

Comment:
what if active uses is admin ?

Reply:
then this will have no real effect. We can add an if-then-else for that. Will do that even if effect is minute

Reply:
done